### DISCUSSION TITLE
yolo5  output as  hls  stream

### DISCUSSION THREAD

**Discussion by samehsy** [ultralytics/yolov5#7345] [Q&A]
Hello . I was  wondering  if  **yolo5**  in  someway  can  output a detected video  stream  as  **hls**  stream or another type of **real-time** streaming  to  view it  in  the  browser, I  know  that yolo5  can  handle  input  as a stream  but  I  need the  output  to  be  the  same,
I hope to find any help or  any type  of  advice  if there is another  way  to  show  the  **Yolo** result in the browser in **real time**.

**Comment by uxfion** [comment]
Do you have any idea to make output detected video stream and view it in the browser real-time?
